lib/device.h

```diff
@@ -150,6 +150,7 @@ metal_device_io_region(struct metal_device *device, unsigned index)
 		: NULL);
 }
 
+extern int metal_linux_get_device_property(struct metal_device *device, const char *property_name, char *output, int len);
 /** @} */
 
```

Comment:
please move this declaration lib/system/linux

Reply:
please change "char *" to "void *" as the output data may not be "char *"

Reply:
I couldn't  find device.h counterpart for lib/system/linux,  Can i create it and move the declaration there?

Reply:
as this is for Linux only for now. keep it in linux/device.c only, and in the drivers, uses "extern" declaration as we don't expect to expose this API to users yet.